bom/runtime/pom.xml

```diff
@@ -2724,6 +2725,26 @@
             </dependency>
 
             <!-- Spring -->
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-spring-di-api</artifactId>
+                <version>${quarkus-spring-api.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-spring-web-api</artifactId>
+                <version>${quarkus-spring-api.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-spring-security-api</artifactId>
+                <version>${quarkus-spring-api.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-spring-data-jpa-api</artifactId>
+                <version>${quarkus-spring-api.version}</version>
+            </dependency>
             <dependency>
                 <groupId>org.springframework</groupId>
                 <artifactId>spring-context</artifactId>
```

Comment:
Shouldn't the dependency below be removed from the BOM? We don't expect Spring artifacts anymore, do we?